composer.json

```diff
@@ -0,0 +1,18 @@
+{
+    "name": "redmonks/magento2-module-quick-email-template-editor",
```

Comment:
change name to "redmonks/module-quick-email-template-editor"

Reply:
Fixed